code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Drugs.dm

```diff
@@ -317,10 +381,94 @@
+	var/mob/living/carbon/human/H = M
+	if(istype(H) && (H.species.flags & NO_BLOOD))
+		return
+	M.add_chemical_effect(CE_PULSE, -1)
+
+	var/power = (M.chem_doses[type] + REAGENT_VOLUME(holder, type))/2 //Larger the dose and volume, the more affected you are by the chemical.
+
+	M.add_chemical_effect(CE_PAINKILLER, 5 + round(power,5))
+
+	if(power < 15)
+		if(prob(3))
+			to_chat(M, SPAN_GOOD(pick("Your anxieties fade away.","Just be yourself - stop caring about what others think.","Everything will be alright...","The world feels so much more vibrant!","You feel relaxed.")))
+
+	if(power > 15)
+		M.drowsiness = min(20,max(M.drowsiness,power - 5))
+		var/nutrition_percent = M.nutrition/M.max_nutrition
+		M.adjustNutritionLoss(power*removed*nutrition_percent)
+		var/hunger_strength = 1 - nutrition_percent
+		if(prob( round(hunger_strength*removed) ))
+			to_chat(M, SPAN_NOTICE(pick("You could really go for some munchies.","You feel the need to eat more.","You crave chips for some reason.","You kind of really want pizza.","Some cosmic brownies would be nice.")))
+		if(prob(3))
+			to_chat(M, SPAN_WARNING(pick("It's a little bit difficult to focus.","You feel a bit lethargic."," It's kinda hard to pay attention to what you're doing.", "Am I forgetting something?", "What was I doing again?")))
+
+/decl/reagent/ambrosia_extract/overdose(var/mob/living/carbon/M, var/alien, var/removed = 0, var/scale = 1, var/datum/reagents/holder)
+	. = ..()
+	M.hallucination = max(M.hallucination, 30 * scale)
+
+/decl/reagent/joy
+	name = "Joy"
+	description = "An expensive and illegal drug often abused by those who find no other means to numb their physical and mental pains. A Joy addict is a truly sad sight."
+	reagent_state = LIQUID
+	color = "#FFFFFF"
+	metabolism = REM*0.1
+	overdose = 5
+	od_minimum_dose = 2
+	taste_description = "tranquility"
+
+/decl/reagent/joy/initial_effect(var/mob/living/carbon/human/M, var/alien, var/holder)
+	to_chat(M, SPAN_GOOD(pick("You feel a numbing sensation spread from within you.", "A numbing sensation builds within you.", "Everything will be okay... just relax.")))
+
+/decl/reagent/joy/affect_blood(var/mob/living/carbon/M, var/alien, var/removed, var/datum/reagents/holder)
+	M.add_chemical_effect(CE_PAINKILLER, 100)
+	M.add_chemical_effect(CE_NEUROTOXIC, 4*removed)
+	M.add_chemical_effect(CE_PACIFIED, 1)
+	M.eye_blurry = max(M.eye_blurry, 30)
+	M.drowsiness = max(M.drowsiness, 10)
+	M.make_dizzy(15)
+	if(M.chem_doses[type] < 1)
+		if(prob(3))
+			to_chat(M, SPAN_GOOD(pick("Stress was an inconvenience that you are now free of.", "You feel somewhat dettached from reality.", "You can feel time passing by and it no longer bothers you.", "You feel so incredibly relaxed.", "You haven't felt this care-free since you were a child...", "Why can't it always be like this?", "You're watching yourself from afar - dettached from your physical body.")))	
```

Comment:
```suggestion
			to_chat(M, SPAN_GOOD(pick("Stress was an inconvenience that you are now free of.", "You feel somewhat dettached from reality.", "You can feel time passing by and it no longer bothers you.", "You feel so incredibly relaxed.", "You haven't felt this care-free since you were a child...", "Why can't it always be like this?", "You're watching yourself from afar - detached from your physical body.")))	
```